package.json

```diff
@@ -15,20 +18,10 @@
     "rxjs": "^7.8.0",
     "typescript": "^5.0.2"
   },
-  "devDependencies": {
-    "@types/jest": "^29.2.4",
-    "jest": "^29.5.0",
```

Comment:
this reverts some of the libraries you added on the make_component branch. can restore maybe some of them but I just use the test libraries already built into create-react-app

Reply:
yeah, I had a little bit of kitchen sinking going on. I looked at the deps for the other js packages and just grabbed all of the jest related things.